dz_s4/zadacha_6_from_seminar.py

```diff
@@ -0,0 +1,20 @@
+# Функция получает на вход список чисел и два индекса.
+# ✔ Вернуть сумму чисел между между переданными индексами.
+# ✔ Если индекс выходит за пределы списка, сумма считается
+# до конца и/или начала списка.
+
+def func(num_list, ind1, ind2):
+    result = 0
+    start_ind = 0
+    end_ind = len(num_list)
+    if 0 <= ind1 <= end_ind:
+        start_ind = ind1
+    if 0 <= ind2 <= end_ind:
+        end_ind = ind2
+    for i in range(start_ind, end_ind):
+        result += num_list[i]
+    return result
+
+
+res_sum = func([1, 2, 3, 4, 6, 0], -2, 8)
```

Comment:
Есть ли индексы будут идти обратном порядке, например, так:
```suggestion
res_sum = func([1, 2, 3, 4, 6, 0], 2, 1)
```
То в результате всегда будет ноль. Но в условии не сказано, в каком порядке они должны идти индексы. Просто два индекса и все. Попробуйте это учесть